### ISSUE TITLE
Minimal API version needs update to 0.79.4

### ISSUE THREAD

**Issue by KevynPM** [mtaheij/PoGo-UWP#184]
Pokemon Go has updated their minimal API version to 0.79.4. Our hasher is at version 0.79.3. The game will not connect to Pokemon Go but instead displays an error notice and then closes to protect the trainer account.

Device info
------------------
- Phone type: Lumia 640
- App version: 1.4.0

Steps to reproduce
----------------
Run the app 

Actual behavior
-----------------
Displays error message: "The version of the hasher (0.79.3) does not match the minimal API version of Pokemon Go (0.79.4). Set 'Configuration. IgnoreHashVersion' to true it you want to disable the version check. To protect your trainer account, we will NOT send any requests to the game servers, but will close now."

Expected behavior
---------------
App should connect.

Other information
---------------
Sad face :'(

**Comment by Furtif** [comment]
fix at https://github.com/mtaheij/PoGo-UWP/pull/179

**Comment by KevynPM** [comment]
@Furtif Do you know when a new release will be available containing the fix?

**Comment by Furtif** [comment]
Look side discord maybe, or wait to new release 0.79.4 is now forced, maybe admin of this git repo make a new release for it...

**Comment by KevynPM** [comment]
The last release worked until yesterday. Now it says 0.79.4 is required.

**Comment by Furtif** [comment]
yes because api as changed and forced ask @mtaheij for new release

**Comment by KevynPM** [comment]
@mtaheij Could you make a new release for 0.79.4 please? Thank you.

**Comment by Furtif** [comment]
https://discord.gg/yyzF5eW maybe

**Comment by Furtif** [comment]
https://github.com/mtaheij/PoGo-UWP/pull/190
